apps/web/components/message.tsx

```diff
@@ -29,74 +31,85 @@ export const Message = ({
   author,
   createdAt,
   attachments,
+  reply,
 }: MessageProps) => {
   const createdAtTimes = buildPostTimeValues(createdAt)
 
   return (
-    <div id={`message-${snowflakeId}`} className="group ">
-      <div className="flex flex-row items-start pointer-events-none [&>*]:pointer-events-auto">
-        <div className="flex justify-start items-start w-[50px] sm:w-[60px] shrink-0">
-          {isFirstRow ? (
-            <img
-              src={author.avatarUrl}
-              alt="Avatar"
-              className="w-10 h-10 rounded-full"
-            />
-          ) : (
-            <time
-              className="hidden self-center text-center group-hover:flex w-full items-center text-xs opacity-70"
-              dateTime={createdAtTimes.iso}
-              title={createdAtTimes.tooltip}
-            >
-              <DisplayLocalTime short dateStr={createdAt.toISOString()} />
-            </time>
-          )}
-        </div>
+    <MessageWrapper snowflakeId={snowflakeId}>
+      <div id={`message-${snowflakeId}`} className="group">
+        <div className="pointer-events-none flex w-full grow-0 flex-col items-stretch justify-start gap-1 p-2 [&>*]:pointer-events-auto">
```

Comment:
```suggestion
        <div className="pointer-events-none flex w-full grow-0 flex-col items-stretch justify-start gap-1 md:gap-0 p-2 [&>*]:pointer-events-auto">
```

I don't think gap is needed for desktop. I was just talking about  adding it for mobile only